release.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  branches: ['master'],
+  plugins: [
+    '@semantic-release/commit-analyzer',
+    '@semantic-release/release-notes-generator',
+    '@semantic-release/changelog',
+    '@semantic-release/npm',
+    '@semantic-release/git',
+  ],
```

Comment:
you need to build the package before publishing, and then publish from the `dist` folder

```suggestion
  plugins: [
    '@semantic-release/commit-analyzer',
    '@semantic-release/release-notes-generator',
    '@semantic-release/changelog',
    ['@semantic-release/exec', {prepareCmd: './build.sh'}],
    ['@semantic-release/npm', {pkgRoot: 'dist'}],
    ['@semantic-release/git', {assets: ['package.json']}],
  ],
```

add this to the package.json scripts:
```
"scripts": {
  ...
  "postversion": "cp -r package.json .."
}
```

https://github.com/semantic-release/exec
https://github.com/semantic-release/npm#examples

Reply:
Appreciate the documentation, editing right now to support releasing shisell from `dist`. 
Why use `['@semantic-release/exec', {prepareCmd: './build.sh'}]` instead of `prepack` command?

Reply:
because the `prepack` command runs after you run `run publish` and before the package is actually packaged. we need to run the `build` script _before_ we run `run publish` because the `run publish` command needs to run inside the `dist` folder that the build script creates